src/IDBStore.js

```diff
@@ -73,41 +73,39 @@ var IDBStore = {
         }
         return callback(null, result);
       };
-      req.onerror = (error) => {
-        callback(error);
-      };
+      req.onerror = callback;
     });
   },
   setFile(dbName, id, data, callback) {
     IDBStore.getStore(dbName, 'readwrite', (err, store) => {
       if (err) return callback(err);
       var req = store.put(data, id);
       req.onsuccess = (event) => callback();
```

Comment:
As mentioned in previous comment, you can probably remove `event` here unless you want to keep it for clarity.